### PR TITLE
Fix special chars in config & allow keylime-selinux

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: secrets/packit/prod/packit-service.yaml.j2 [hunk 1]
```diff
@@ -31,7 +32,7 @@ command_handler_pvc_volume_specs:
   - path: /repository-cache
     pvc_from_env: SANDCASTLE_REPOSITORY_CACHE_VOLUME
     read_only: true
-# [TODO]: Switch to ‹aws-ebs› during migration of prod to MP+
+# [TODO]: Switch to <aws-ebs> during migration of prod to MP+
```
Comment (mfocko) [inline secrets/packit/prod/packit-service.yaml.j2:35]: Why is that an issue »now« and not before? :eyes:

**Review by mfocko — APPROVED** [review]

File: secrets/packit/prod/packit-service.yaml.j2 (hunk 1 above)
Comment (mfocko) [reply secrets/packit/prod/packit-service.yaml.j2:35]: Oh I know… the “OpenShift Dashboard”, we don't include the comments when it's run via `make deploy`.

**Comment by majamassarini** [comment]
regate

**Comment by majamassarini** [comment]
recheck

**Pull request merged**